x-pack/plugins/apm/server/lib/helpers/convert_ui_filters/get_parsed_ui_filters.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Logger } from 'src/core/server';
+import { UIFilters } from '../../../../typings/ui_filters';
+
+export function getParsedUiFilters({
```

Comment:
perhaps this should be used here as well: https://github.com/elastic/kibana/blob/ebc0b86697e2c1d2594225a155727eb8d63948bd/x-pack/plugins/apm/server/lib/helpers/setup_request.ts#L26